### PR TITLE
Oh god, he can't hear us, he's got airpods on

### PR DESCRIPTION
**Pull request by alsoandanswer**
"Adds earbuds. Also makes earphones and headphones colorable."

### CODE REVIEW

**Comment by alsoandanswer** [comment]
!review

**Review by SleepyGemmy — CHANGES_REQUESTED** [review]
make the items you touch into contained sprites, please.

**Review by SleepyGemmy — APPROVED** [review]
suggested changes addressed.

File: code/modules/clothing/ears/earmuffs.dm [hunk 1]
```diff
@@ -1,23 +1,26 @@
...
-/obj/item/clothing/ears/earmuffs/headphones
-	name = "headphones"
-	desc = "Unce unce unce unce."
-	var/headphones_on = 0
-	icon_state = "headphones"
-	item_state = "headphones"
+/obj/item/clothing/ears/earmuffs/earphones
+	name = "earphones"
+	desc = "A pair of wired earphones. Good luck trying to find a headphone jack."
+	desc = "Has a tendency to snag itself on whatever furniture it can find at the most inopportune of times."
```
Comment (NonQueueingMatt) [inline code/modules/clothing/ears/earmuffs.dm:15]: Double definition

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**